### ISSUE TITLE
Check for existence of Cache-Control header before attempting to read it.

### ISSUE THREAD

**Issue by tomayac** [micmro/PerfCascade#111]
I noticed a problem when testing this project with http://www.tagesschau.de/ and managed to track down the issue to [heuristics.ts#L58](https://github.com/micmro/PerfCascade/blob/master/src/ts/helpers/heuristics.ts#L58), which currently reads…

```typescript
return getHeader(headers, "Cache-Control").indexOf("no-cache") > -1
```

…but actually should have an existence check prior to attempting to look for the "no-cache" string like so…

```typescript
return getHeader(headers, "Cache-Control") && getHeader(headers, "Cache-Control").indexOf("no-cache") > -1
```

I tested this manually locally in my ```src/dist/perf-cascade.js``` file with success.

**Comment by tobli** [comment]
Thanks for the report, this was mistakenly introduced in d21f346 (released with version 0.2.17). I'm looking into a fix that I hope to PR soon.

**Comment by tomayac** [comment]
Awesome, thanks 👍🏿 ! Could you publish 0.2.18 to npm, please? It's still on 0.2.17.

**Comment by micmro** [comment]
Oh must have accidentally screwed up the npm release - will do tonight (Korean time)

**Comment by tobli** [comment]
This fix was merged post 0.2.18, so we need a 0.2.19 for this

**Comment by micmro** [comment]
Ok just released `0.2.19`
